tools/dockerbuild/docker/Dockerfile-native

```diff
@@ -0,0 +1,22 @@
+####
+#
+# This Dockerfile is used in order to build a container that runs a Quarkus application in native mode.
+#
+# Before building the container image, build the Quarkus application using the "native" packaging.
+# For example:
+#    ./gradlew :nessie-quarkus:quarkusBuild -Pnative
+#
+#
+#
+###
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+WORKDIR /work/
+RUN chown 1001 /work \
+    && chmod "g+rwX" /work \
+    && chown 1001:root /work
+COPY --chown=1001:root build/*-runner /work/application
+
+EXPOSE 8080
+USER 1001
+
+CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]
```

Comment:
Why not `ENTRYPOINT`?

Do we want to support custom exec's into this container?